.github/workflows/release.yaml

```diff
@@ -110,6 +122,20 @@ jobs:
           file: config/.release/install.yaml
           asset_name: install.yaml
           tag: ${{ github.ref }}
+      
+      - name: Push yaml manifest into an OCI registry
+        env:
+          COSIGN_EXPERIMENTAL: 1
+        run: |
+            cosign login --username ${GITHUB_ACTOR} --password ${{ secrets.CR_PAT }} ghcr.io
+            cd config/.release/ && \
```

Comment:
I did this workaround because Flux CLI (in the meantime) is not accepting files in the `--path` parameter. We (w/ @stefanprodan) fixed that, but this fix has not been released yet, so once we release that, we'll be able to accept files in that parameter and will use `--path=install.yaml` directly.

I added `/tmp` folder into the `.gitignore` folder, PTAL @eddycharly

Reply:
There is no need for this, please undo the `/tmp` change.